bgpd/bgpd.c

```diff
@@ -3077,11 +3035,20 @@ int peer_group_remote_as(struct bgp *bgp, const char *group_name, as_t *as,
 	return 0;
 }
 
-void peer_notify_unconfig(struct peer *peer)
+bool peer_notify_config_change(struct peer_connection *connection)
 {
-	if (BGP_IS_VALID_STATE_FOR_NOTIF(peer->connection->status))
-		bgp_notify_send(peer->connection, BGP_NOTIFY_CEASE,
-				BGP_NOTIFY_CEASE_PEER_UNCONFIG);
+	if (BGP_IS_VALID_STATE_FOR_NOTIF(connection->status)) {
+		bgp_notify_send(connection, BGP_NOTIFY_CEASE, BGP_NOTIFY_CEASE_PEER_UNCONFIG);
```

Comment:
Here it should be BGP_NOTIFY_CEASE_CONFIG_CHANGE, right?

Reply:
fixed